app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/RefreshController.kt

```diff
@@ -309,19 +330,27 @@ class RefreshController @Inject constructor(
         }
     }
 
-    private suspend fun Realm.fetchActivities(scope: CoroutineScope, folder: Folder, previousCursor: String) {
+    private suspend fun Realm.fetchActivities(scope: CoroutineScope, folder: Folder, previousCursor: String): Int {
```

Comment:
We can't be returning meaningless integers like that, use an enum it's much more appropriate

Reply:
It's been updated.